app/Main.hs

```diff
@@ -84,7 +93,7 @@ opts =
 main :: IO ()
 main = do
   canUseColor <- supportsPretty
-  config <- Opt.execParser opts
+  config <- Opt.customExecParser (Opt.prefs Opt.showHelpOnError) opts
```

Comment:
I took me a while to understand what you were doing here.

Apparently you want to display the help on error, which was not initially the case.

I feel that it is orthogonal to this commit. However That's a good idea, so why not.

Reply:
Hum, might be a remnant of me trying to make it work. I agree that I think that this has nothing to do with the issue. I think I left that by mistake